### ISSUE TITLE
Auto generate definition questions and dialogues about translations using Wikidata

### ISSUE THREAD

**Issue by stefangrotz** [LAION-AI/Open-Assistant#2195]
Wikidata Lexemes and the description field that exists for many data objects could be easily used to automatically create short dialogues with a standardized structure like this:

A: Whats the definition of {X}
B The definition is {Y}, but it can also mean {Z}

You can also use other fields in the dataset like synonyms to expand the dataset and you can easily create dialogues about word meanings and translations.

Here is a simple query to give you an impression how the basic data looks:

https://query.wikidata.org/#SELECT%20%3Flexeme%20%3Flemma%20%3Fsense%20%3Fgloss%20WHERE%20%7B%0A%20%20%3Flexeme%20dct%3Alanguage%20wd%3AQ1860%3B%20%23%20English%20language%0A%20%20%20%20%20%20%20%20%20%20wikibase%3Alemma%20%3Flemma%3B%0A%20%20%20%20%20%20%20%20%20%20ontolex%3Asense%20%3Fsense.%0A%20%20%3Fsense%20skos%3Adefinition%20%3Fgloss.%0A%20%20FILTER%28lang%28%3Fgloss%29%20%3D%20%22en%22%29%20%23%20Filter%20only%20English%20definitions%0A%7D%0ALIMIT%2050%0A

**Comment by davidak** [comment]
I think when you add many examples with the same structure, it produces low quality answers. It might be enough to add one example of each type.

**Comment by stefangrotz** [comment]
I agree, the mayority of the training data should be dialogues created by humans.

However, data created like this can cover many area of life and it could be especially useful on dialogues about translations and word meanings.

It is also possible to create 20 variants of each question with different wordings and loop through the variants during the creation script. This would increase the variance at least a little.
